src/Catalyst.Protocol/Transaction/PublicEntry.cs

```diff
@@ -47,8 +47,12 @@ public bool IsValid()
                 return false;
             }
 
-            var hasValidSignature = Signature.IsValid(SignatureType.TransactionPublic);
-            return hasValidSignature;
+            return true;
+
+            // TODO: reconsider signature
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)